Packs/CohesityHelios/Integrations/CohesityHelios/CohesityHelios.yml

```
@@ -0,0 +1,171 @@
+    - name: limit
+      defaultValue: 200
+      default: false
+      description: Limits the number of alerts to return.
+      isArray: false
+      required: false
+    - default: false
+      description: List of comma-separated alert identifiers to filter alerts.
+      isArray: true
+      name: alert_id_list
+      required: false
+    - default: false
+      description: List of comma-separated cluster identifiers to filter alerts.
+      isArray: true
+      name: cluster_id_list
+      required: false
+    - default: false
+      description: List of comma-separated region identifiers to filter alerts.
+      isArray: true
+      name: region_id_list
+      required: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: kOpen
+      description: One or more state values.
+      isArray: true
+      name: alert_state_list
+      required: false
+      predefined:
+      - kOpen
+      - kSuppressed
+      - kResolved
+      - kNote
+    - auto: PREDEFINED
+      default: true
+      defaultValue: kCritical,kWarning
+      description: One or more severity levels.
+      isArray: true
+      name: alert_severity_list
+      required: false
+      predefined:
+      - kCritical
+      - kWarning
+      - kInfo
+    deprecated: false
+    description: Get Cohesity Helios ransomware alerts.
+    execution: false
+    name: cohesity-helios-get-ransomware-alerts
+    outputs:
+    - contextPath: CohesityHelios.RansomwareAlert.alert_id
+      description: Identifier for the ransomware alert.
+      type: String
+    - contextPath: CohesityHelios.RansomwareAlert.occurrence_time
+      description: Name of the detected anomalous object.
+      type: Date
+    - contextPath: CohesityHelios.RansomwareAlert.severity
+      description: Severity of the ransomware alert.
+      type: String
+    - contextPath: CohesityHelios.RansomwareAlert.alert_description
+      description: Description for the ransomware alert.
+      type: String
+    - contextPath: CohesityHelios.RansomwareAlert.alert_cause
+      description: Cause for the ransomware alert.
+      type: String
+    - contextPath: CohesityHelios.RansomwareAlert.anomalous_object_name
+      description: Name of the detected anomalous object.
+      type: String
+    - contextPath: CohesityHelios.RansomwareAlert.anomalous_object_env
+      description: Env of the detected anomalous object.
+      type: String
+    - contextPath: CohesityHelios.RansomwareAlert.anomaly_strength
+      description: Strength of the detected ransomware alert.
+      type: Number
+  - arguments:
+    - default: false
+      description: Anomalous object name to be ignored.
+      isArray: false
+      name: object_name
+      required: true
+    deprecated: false
+    description: Ignore detected anomalous object.
+    execution: false
+    name: cohesity-helios-ignore-anomalous-object
+  - arguments:
+    - default: false
+      description: Anomalous object name to be restored.
```

Following the demo session, in my opinion, it would be much clear if you specify where the user can get the `object_name` from. For example, 
```suggestion
      description: Anomalous object name to be restored. Can be retrieved by running the command cohesity-helios-get-ransomware-alerts.
```